File: prototype/spellslingerer/src/store/modules/planeswalkers.js
import allPlaneswalkers from '../data/planeswalkers';

export default {
  namespaced: true,
  getters: {
    planeswalkers: () => Object.values(allPlaneswalkers).filter(({ id }) => id !== 'Default'),
    planeswalker: () => (planeswalkerId) => allPlaneswalkers[planeswalkerId],
    planeswalkerNames: () => Object.values(allPlaneswalkers).filter(({ id }) => id !== 'Default').map(({ name }) => name),
  },
};

File: api/models/planeswalkers.go
package models

import "api/db"

// Planeswalker is used to craft cards
type Planeswalker struct {
	ID   int64
	Name string
}

// AllPlaneswalkers enumerates all planeswalkers
func AllPlaneswalkers() ([]Planeswalker, error) {
	var allPlaneswalkers []Planeswalker
	err := db.GlobalSqlxDB.Select(&allPlaneswalkers, "SELECT * FROM public.planeswalkers")
	return allPlaneswalkers, err
}
